react/components/FilterOptionTemplate.js

```diff
@@ -29,16 +34,43 @@ const FilterOptionTemplate = ({
   children,
   filters,
   initiallyCollapsed = false,
+  truncateFilters = false,
 }) => {
   const [open, setOpen] = useState(!initiallyCollapsed)
   const handles = useCssHandles(CSS_HANDLES)
+  const [truncated, setTruncated] = useState(true)
+  const shouldTruncate =
+    truncateFilters && filters.length >= MAX_ITEMS_THRESHOLD
 
   const renderChildren = () => {
     if (typeof children !== 'function') {
       return children
     }
-
-    return filters.map(children)
+    return (
+      <>
+        {filters
+          .slice(
+            0,
+            shouldTruncate && truncated ? MAX_TRUNCATED_ITEMS : filters.length
+          )
+          .map(children)}
+        {shouldTruncate && (
+          <button
+            onClick={() => setTruncated(truncated => !truncated)}
+            className={`${handles.seeMoreButton} mt2 pv2 bn pointer c-link`}
+          >
+            <FormattedMessage
+              id={
+                truncated
+                  ? 'store/filter.more-items'
+                  : 'store/filter.less-items'
+              }
```

Comment:
```suggestion
              id="store/filter.more-items"
```

Reply:
in fact, this message is still needed. when opening all filters, the user has the option to truncate again, with the button that displays the message "see less" :))

Reply:
ah I see, I misunderstood the code